docs/generating_the_url_advanced.md

```diff
@@ -428,6 +428,17 @@ It's highly recommended to prefer `cachebuster` option over URL query string bec
 
 Default: empty
 
+#### Keep Meta
+
+```
+stripmeta:%stripmeta
```

Comment:
Still, let's name it `strip_metadata`. The same goes for the `ProcessingOptions` field.